PyInstaller/lib/modulegraph/modulegraph.py

```diff
@@ -133,7 +133,7 @@
     "import sys,types,os; p = os.path.join(sys._getframe(1).f_locals['sitedir'], *('",
     "import sys,new,os; p = os.path.join(sys._getframe(1).f_locals['sitedir'], *('",
     "import sys, types, os;p = os.path.join(sys._getframe(1).f_locals['sitedir'], *('",
-    "import sys, types, os;pep420 = sys.version_info > (3, 3);p = os.path.join(sys._getframe(1).f_locals['sitedir'], *('",
+    "import sys, types, os;pep420 = sys.version_info > (3, 4);p = os.path.join(sys._getframe(1).f_locals['sitedir'], *('",
```

Comment:
This *must not* be changed. These are pattern to match files generated by setuptools. And setuptools (or some version of it out in the wild) may still generate this code.